backend/dev/user.clj

```diff
@@ -22,6 +22,7 @@
    [app.common.types.file :as ctf]
    [app.common.uuid :as uuid]
    [app.config :as cf]
+   [app.db :as db]
```

Comment:
Is this used?

Reply:
Yep, is a `user` ns, is just for make it easy to access db from repl

Reply:
true!